### ISSUE TITLE
Final Report Peer Review (mh2386)

### ISSUE THREAD

**Issue by TheMichaelHu** [LillyanPan/AttritionAnalysis#9]
The goal of this project was to identify which features contribute the most towards the loss of employees. Identifying these key factors would allow companies to take measures to improve employee retention and understand how to be better employers. I really liked this idea because of its focus on training model not to make predictions, but to provide insight on why an event may occur. The group had a relatively clean dataset, but still had to deal with the issue of imbalanced classes. This was done with SMOTE, an oversampling method that goes beyond the topics covered in class, which was awesome. The group also avoided many of the pitfalls associated with imbalanced classes via looking at recall in addition to train/test error.
In terms of criticisms and suggestions, I noticed a few typos in the report such as calling their svm classifier a Support Vector Clustering and stating that they used l2 regularization to produce sparse weights when they probably meant l1 (actually that seemed like less of a typo and more of a misunderstanding). Also, I found Figure 2 to be a bit difficult to interpret without a legend, although I believe it was implied that the color encodings from Figure 3 were that same as those for Figure 2.
All in all I think this was a very interesting project with useful results. Now I will be suspicious of ulterior motives if my employer ever starts pressuring me to get a girlfriend; it'll probably be so I'm less likely to quit!

**Comment by LillyanPan** [comment]
Thank you for the feedback! One thing to note is that Support Vector Clustering is not a typo. We used SVC for classification, which is a subset of SVM– SVM is an algorithm, while SVC is a model.

**Comment by TheMichaelHu** [comment]
I realize this is overly nit picky, but you're saying you used support vector clustering (an unsupervised model) , not [support vector classification](http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html)? I wasn't expecting unsupervised learning in a project so I assumed you meant classification instead of clustering; it's wicked cool if you used clustering though!
